pkg/azurefile/controllerserver.go

```diff
@@ -117,13 +117,14 @@ func (d *Driver) DeleteVolume(ctx context.Context, req *csi.DeleteVolumeRequest)
 	}
 
 	if err := d.ValidateControllerServiceRequest(csi.ControllerServiceCapability_RPC_CREATE_DELETE_VOLUME); err != nil {
-		return nil, fmt.Errorf("invalid delete volume req: %v", req)
+		return nil, status.Errorf(codes.InvalidArgument, "invalid delete volume request: %v", req)
 	}
 
 	volumeID := req.VolumeId
 	resourceGroupName, accountName, fileShareName, err := getFileShareInfo(volumeID)
 	if err != nil {
-		return nil, err
+		klog.V(2).Infof("azure file(%s) under rg(%s) account(%s) volumeID(%s) is invalid, returning with success: %v", fileShareName, resourceGroupName, accountName, volumeID, err)
```

Comment:
`klog.Errorf("getFileShareInfo(%s) in DeleteVolume failed with error: %v`, volumeID)`